proto/delivery/delivery.proto

```diff
@@ -211,4 +211,10 @@ message Insertion {
   reserved 16;
   reserved 17;
   reserved 18;
+
+  // Optional. Ranking (if known) of this insertion from the retrieval system.
+  optional uint64 retrieval_rank = 19;
```

Comment:
why do we we need both?  isn't rank just integer score?  would these be a oneof relationship?

Reply:
The way to think about score is the Elasticsearch score, and the way to think about rank is where the marketplace would position the content (for which the score may be one but not the only factor). Suggest discussing with @andrewdyates for more details.